server/utils/users.js

```diff
@@ -1,18 +1,20 @@
 const Conversation = require('../models/Conversation');
 const users = [];
-const addUser = async({ id, userProfileId, conversationId }) => {
-   
-  const conversation=await Conversation.findById({_id:conversationId})
-    
-   if(userProfileId!==conversation.senderProfileId||userProfileId!==conversation.recieverProfileId)
-       return { error: 'Wrong userProfileId' };
-   
- 
-  const user = { id, userProfileId, conversationId };
-
-  users.push(user);
-
-  return { user };
+const addUser = async ({ id, userId ,conversationId}) => {
+  const conversation = await Conversation.findById({ _id: conversationId });
```

Comment:
`findById` method doesn't require this object. Just pass the `id` as an argument.